RELEASE/scripts/autoscend.ash

```diff
@@ -5231,6 +5224,7 @@ void auto_begin()
 	backupSetting("battleAction", "custom combat script");
 
 	backupSetting("choiceAdventure1107", 1);
+	backupSetting("choiceAdventure330", 1);
```

Comment:
comment

Reply:
To clarify what I am doing here.
We had code that did this
```
backup setting
adv 1 turn
restore setting
return true
```
this both assumed success and was unnecessary flip flopping on the setting compared to just backing it up once at the begining of the run and then restoring it with all other restored settings in the end.
I replaced it with `if(autoAdv(loc)) return true;` and this setting at the beginning of the run

Although to be honest I am not sure why we are backing and restoring some NC choice adventures while others we just change on the spot without worrying about changing them back.

Reply:
comment added